client/client.go

```diff
@@ -203,6 +205,22 @@ func (c *client) checkStreamTimeout(loopCtx context.Context, cancel context.Canc
 	}
 }
 
+func (c *client) GetMemberInfo(ctx context.Context) ([]*pdpb.Member, error) {
+	start := time.Now()
+	defer func() { cmdDurationGetMemberInfo.Observe(time.Since(start).Seconds()) }()
+
+	ctx, cancel := context.WithTimeout(ctx, c.timeout)
+	resp, err := c.leaderClient().GetMembers(ctx, &pdpb.GetMembersRequest{})
```

Comment:
Do we need a request header?

Reply:
I tried to get the result of correct request without request header, but to be safe, I can add header in it

Reply:
> Do we need a request header?

I tried to get the result of correct request without request header, but to be safe, I can add header in it